config.toml

```diff
@@ -1,3 +1,4 @@
+baseURL = "https://ipfs.io/blog"
```

Comment:
How's that: https://ipfs.io/ipfs/QmdSwmRVWDDpDoBYXJcjHavF1jFhvkvaWwGAXeqAQjS1ii/

I've just changed the URLs to be relative when they should be.

Sorry I didn't see your comment in time, but is that acceptable?